doc/ug/getting_started_fpga.md

```diff
@@ -151,6 +151,8 @@ $ cd $REPO_TOP
 $ /tools/openocd/bin/openocd -s util/openocd -f board/lowrisc-earlgrey-nexysvideo.cfg
 ```
 
+Note that you must use the RISC-V fork of OpenOCD avaliable from: https://github.com/riscv/riscv-openocd.
```

Comment:
in our installation guide, we do talk about installing the riscv fork.  Perhaps we should add a link to reference there?

Reply:
You mean point to the installation guide?

Lots of people don't read the full installation guide and just try to use their/mainline OpenOCD (me included). Having it explicitly here makes it apparent that upstream doesn't work yet.

Reply:
o no i meant have this language here and also point to the installation guide. I think that could be helpful hint to first time users that there may be other setup tidbits to look over. Just a thought.

Reply:
Ah, good point. I just pushed an update.